pkgs/development/python-modules/openai-triton/default.nix

```diff
@@ -71,7 +74,10 @@ buildPythonPackage rec {
     zlib
   ];
 
-  propagatedBuildInputs = [ filelock ];
+  propagatedBuildInputs = [
+    filelock
+    setuptools
```

Comment:
Please add a comment, ideally with a permalink. Otherwise the next editor's first prompt might be to remove this as it looks like a mistake